src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -310,6 +313,16 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
           }
         }
 
+        this.logService.createFeedbackSessionLog({
+          courseId: this.courseId,
+          studentEmail: this.loggedInUser,
```

Comment:
Let's log down the fsname too

Reply:
Also, are you sure this is the best place to track the student access? What if the user is using a `regKey`?

Reply:
Wouldn't the `loadFeedbackSession` function be called regardless of which auth the user uses?

Reply:
It is true that the method will be called if auth succeeds. But the thing is the `loggedInUser` may not contain the email of the student. I was thinking a safer place to put it would be in the method `loadPersonName`